manual/server/build-instructions.md

```diff
@@ -1,13 +1,27 @@
 # Build instructions
 
-This guide tells how to build the server.
+This guide tells you how to build the server.
```

Comment:
Finally an unbiased reader can judge if this information is useful :D

Reply:
:) `tells` can't be used without an object ("whom is what told?"). You could use `describe` and drop the `you` I edited in there, it may feel slightly better.

Reply:
I just meant the general fact that someone except me has read this manual ;) The missing "you" is obvious :sweat_smile: